src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/TseViewApplicationHelper.java

```diff
@@ -96,8 +93,8 @@ public static DynamicFixedListType populateOpenOrClosedApplications(CaseData cas
         boolean selectedClosed = CLOSED_STATE.equals(caseData.getTseViewApplicationOpenOrClosed());
 
         return DynamicFixedListType.from(caseData.getGenericTseApplicationCollection().stream()
-                .filter(o -> selectedClosed ? o.getValue().getStatus().equals(CLOSED_STATE)
-                        : !o.getValue().getStatus().equals(CLOSED_STATE))
+                .filter(o -> selectedClosed ? CLOSED_STATE.equals(o.getValue().getStatus())
+                        : !CLOSED_STATE.equals(o.getValue().getStatus()))
```

Comment:
Would this also work? 
`.filter(o -> selectedClosed == CLOSED_STATE.equals(o.getValue().getStatus()))`

That way, the item will only get included if `selectedClosed` is true and getStatus() is CLOSED_STATE or if `selectedClosed` is false and getStatus() is not equal to CLOSED_STATE